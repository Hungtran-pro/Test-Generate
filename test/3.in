9
10
26
13
17
32
29
25
12
27
