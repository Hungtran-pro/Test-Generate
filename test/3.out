275186727
824482804
518869573
787652498
811074431
392693967
304394701
658214825
269420915
